README.md

```diff
@@ -247,7 +247,7 @@ To contribute to Rust, please see [CONTRIBUTING](CONTRIBUTING.md).
 Most real-time collaboration happens in a variety of channels on the
 [Rust Discord server][rust-discord], with channels dedicated for getting help,
 community, documentation, and all major contribution areas in the Rust ecosystem.
-A good place to ask for help would be the #help channel.
+A good place to ask for help would be the #contribute channel.
```

Comment:
Yeah, I guess that most accurately we would want to point people to either Discord or Zulip, depending on what sort of contribution they have in mind, though some groups (e.g., embedded) are elsewhere.

Reply:
At least adding a link to Zulip seems good, though I suppose the ideal would be to list out the various teams at each spot, and I'm a bit reluctant to add that info here for fear of it getting outdated. But we should have some central resource, *probably* organized through the teams repo?